diktat-rules/src/test/resources/test/smoke/save.toml

```diff
@@ -1,8 +1,13 @@
 [general]
-execCmd="java -jar ktlint-snap.jar -R diktat-1.0.0-rc.2.jar"
+execCmd="java -jar ktlint -R diktat-1.1.0.jar"
```

Comment:
Shouldn't we use current snapshot of diktat here?

Reply:
Yes. I tested on different versions. I will change the save.toml later

Reply:
can we change it to `diktat-*` ? does `ktlint` support it?